app/services/verification_request_checker.rb

```diff
@@ -0,0 +1,13 @@
+class VerificationRequestChecker
```

Comment:
I am not sure if you need a service class for checker, it doesn't seem there is a lot of logic and all the functions are just based on the results